app/controllers/peer_reviews_controller.rb

```diff
@@ -63,13 +63,13 @@ def destroy
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_peer_review
-      @peer_review = PeerReview.find(params[:id])
-    end
```

Comment:
Eh, I think this is actually more consistent with the style in other files. I wouldn't have put it in here, but you can leave it.